frontend/src/Component/Share/Navbar/Sidebar.jsx

```
@@ -1,56 +1,100 @@
                 {role === "student" && (
                     <>
                         <SidebarLink to="/student/dashboard" icon={HomeIcon} text="Dashboard" active={location.pathname.startsWith("/student/dashboard")} />
-                        <SidebarLink to="/student/courses" icon={BookOpenIcon} text="Courses" active={location.pathname.startsWith("/student/courses")} />
+
+                        {/* Student Courses with Sub-Options */}
+                        <button
+                            className={`w-full flex items-center justify-between py-2 px-3 rounded-md transition ${
+                                studentCoursesOpen ? "bg-gray-300 text-gray-800 font-semibold" : "text-gray-300 hover:bg-gray-800"
+                            }`}
+                            onClick={() => setStudentCoursesOpen(!studentCoursesOpen)}
+                        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)